scripts/export_database.pl

```diff
@@ -518,14 +521,17 @@ sub sanitize_field_content {
 
 	# only overwrite previous dump if the new one is bigger, to reduce failed runs breaking the dump.
 	my $csv_size_old = (-s $csv_filename) // 0;
-	my $csv_size_new = (-s "$csv_filename.temp") // 0;
+	# Sort lines by code, except header line
+	system("(head -1 $csv_filename.temp && tail -n +2 $csv_filename.temp | sort) > $csv_filename.temp2");
```

Comment:
Might be more readable using one more parenthesis:

```suggestion
	system("(head -1 $csv_filename.temp && (tail -n +2 $csv_filename.temp | sort)) > $csv_filename.temp2");
```